f8a_worker/workers/stackaggregator_v2.py

```diff
@@ -117,12 +118,34 @@ def aggregate_stack_data(stack, manifest_file, ecosystem, deps):
         "packages": license_score_list
     }
 
+    _logger = logging.getLogger(__name__)
+
     try:
         license_req = get_session_retry().post(license_url, data=json.dumps(payload))
         resp = license_req.json()
-        stack_license = [resp.get('stack_license')]
+        
+        stack_license_exists = resp.get("stack_license_exists") 
+        package_with_unknown_license_exists = resp.get("package_with_unknown_license_exists")
+        stack_license_conflict_exists = resp.get("stack_license_conflict_exists")
+        package_license_conflict_exists = resp.get("package_license_conflict_exists")
+        packages_with_license_conflict = resp.get("packages_with_license_conflict")
+        recommended_stack_licenses = resp.get("recommended_stack_licenses")
+        stack_license_conflict_packages = resp.get("stack_license_conflict_packages")
+        stack_license_outlier_packages = resp.get("stack_license_outlier_packages")
+
     except:
-        stack_license = [None]
+        _logger.error("Error getting data from Stack License API")
+
+        #Construct default response if API has issues
```

Comment:
Just a design question - do we want to have this default response here? What about reporting error and letting user to try again?